docs/UserGuide.md

```diff
@@ -267,6 +267,10 @@ of tags is not cumulative.
 
 Acceptable Values: Refer to [Argument Summary](#argument-summary).
 
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
+As your list of clients might be long, you can consider using the [Find](#locating-clients-by-name-financial-plan-andor-tag--find) command first to filter the list before editing details of your client.
```

Comment:
the indexing will change tho?

Reply:
The edit command index is based on the filter list